distribution/std-lib/Standard/src/Image/Data/Histogram.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
+
+polyglot java import org.enso.image.data.Histogram as Java_Histogram
+
+type Histogram
+
+    type Histogram channel data
+
+    ## Convert histogram data to Json.
+    to_json : Json
+    to_json =
+        bins = Json.from_pairs [["bins", this.data]]
+        Json.from_pairs [["data", bins]]
+
+## Create a histogram for a specified channel from the image.
+calculate image channel =
```

Comment:
Missing a signature. Again, why not define this as an extension on the `Image` type?